src/libvirt-dbus.js

```diff
@@ -1305,6 +1306,15 @@ export function attachIface({ connectionName, vmId, mac, permanent, hotplug, sou
     return attachDevice({ connectionName, vmId, permanent, hotplug, xmlDesc });
 }
 
+export function attachHostDevice({ connectionName, vmName, live, dev }) {
+    const source = getNodeDevSource(dev);
+
+    return cockpit.script(
+        `virt-xml -c qemu:///${connectionName} ${vmName} --add-device --hostdev ${source}` + (live ? " --update" : ""),
```

Comment:
As you already use a format string, please use it for the `--update` part as well. Don't mix \` and +.